try:
    n = int(input())
    print(n)
except Exception as e:
    print(e)
